orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/templates/orc8r-values-1.3.yaml.j2

```diff
@@ -84,7 +84,6 @@ nms:
   - name: artifactory
   magmalte:
     env:
-      api_host: api.{{ orc8r_domain }}
```

Comment:
Just checking: is this really what you want? Because we're only changing these values for the latest helm charts, whereas this deployment values.yaml seems to be for v1.3 ?

Reply:
This is intentional and bare-metal-ansible is my sub-project as the only contributor. This removes the requirement for api.DOMAIN to be routable from inside k8s pods and will instead use internal k8s DNS resolution

Reply:
Cool sounds good, up to you. Just need to rebase onto latest master to fix merge conflicts, then we can approve+merge